script/Arcueid/Day 10/10_00_ARC10_6.txt

```diff
@@ -200,7 +200,7 @@ I get the feeling that this boy definitely knows what I should be doing from her
 -- Page 686, Offset 26114.
 -- 「死ね、ブタ。テメェ俺の後ろでＳＰだろうが。ベンチに座んのはオレだけだ。トオノはそこまで気を許しちゃいねえからな、どうせそこで立ちん坊だろ？」
 // Don't think SP is going to be understood by the audience directly here
-"Kill yourself, fatso.
+"Kill yourself, Fatso.
```

Comment:
Does this count as a name? Not sure on that.

Reply:
It's a nickname, I found it capitalized in 

> "Oi, Fatso Noel."

what do you think?